### DISCUSSION TITLE
Can poetry colors be disabled?

### DISCUSSION THREAD

**Discussion by martinda** [python-poetry/poetry#8838] [General] [answered]
This is similar to #3689 but my need is to disable colors completely.

I tried `--no-ansi`, but this removes useful formatting.

I tried `NO_COLOR=true poetry install` and `PY_COLORS=0 poetry install` but those do not disable colors. This was based on the terminalwriter code, unfortunately I could not find source other than https://py.readthedocs.io/en/latest/_modules/py/_io/terminalwriter.html (which appears to be what poetry uses).

I use:
```
Poetry version 1.1.12
Python 3.10.12
```

**Comment by edgarrmondragon** [comment, marked as answer]
`NO_COLOR=1 poetry install` works as expected for me with the latest Poetry (1.7.1), so perhaps all you need to do is upgrade?
